### PR TITLE
fix_pix_3.37.0 

### PR DESCRIPTION
**Pull request by hgaldino**
## Description
In few word, describe what’s your Pull Request about.

### Issue involved
Fixes #<issue>

### Checklist:
Before sending your contribution, please specify the following: 

- [ ] This request modifies code snippets. 
- [ ] The contribution aligns with the information stated in the repository wiki.

- [ ] Contribution incorporates an article not included until now, which must be added to indexes as a new section. 
- [ ] Contribution includes new features. 
- [ ] New features were communicated in changelog.
- [ ] Requires translations.

### CODE REVIEW

File: guides/online-payments/checkout-api/receiving-payment-by-pix.en.md [hunk 1]
```diff
@@ -0,0 +1,335 @@
+----[mlb]----
+[TXTSNIPPET][/guides/snippets/test-integration/pix-intro]
+
+After registering the Pix key, follow the documentation to perform the integration or, if you prefer, [see on Github](https://github.com/mercadopago/pix-payment-sample) for examples of how to add Pix payments by integrating with Checkout Transparente.
+
+## Configure a payment with Pix
+
+After [capturing the data for payment](#bookmark_data_capture_for_payment) in the form, to start getting Pix payments you will need to dend the buyer's email address, document type and number, method of payment and amount.
```
Comment (romisoto) [inline guides/online-payments/checkout-api/receiving-payment-by-pix.en.md:8]: dend = send ??

File: guides/online-payments/checkout-api/receiving-payment-by-pix.es.md [hunk 2]
```diff
@@ -0,0 +1,331 @@
+----[mlb]----
+[TXTSNIPPET][/guides/snippets/test-integration/pix-intro]
+
+Después de registrar la clave Pix, sigue la documentación para realizar la integración o, si lo prefieres, [la puedes ver en Github](https://github.com/mercadopago/pix-payment-sample) para ver ejemplos de cómo agregar pagos Pix por integrarse con Checkout API.
```
Comment (romisoto) [inline guides/online-payments/checkout-api/receiving-payment-by-pix.es.md:4]:
```suggestion
Después de registrar la clave Pix, sigue la documentación para realizar la integración o, si lo prefieres, [accede a Github](https://github.com/mercadopago/pix-payment-sample) para ver ejemplos de cómo agregar pagos Pix por integrarse con Checkout API.
```

File: guides/online-payments/checkout-api/receiving-payment-by-pix.es.md [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+            "neighborhood": "Bonfim",
+            "city": "Osasco",
+            "federal_unit": "SP"
+        }
+      }
+    }'
+```
+]]]
+
+La respuesta va a mostrar el estado pendiente del pago y toda la información que necesitas para mostrar al comprador. El valor `transaction_data` te brindará los datos para disponibilizar la opción de pago a través de un código QR. Vas a encontrar los siguientes atributos:
```
Comment (romisoto) [inline guides/online-payments/checkout-api/receiving-payment-by-pix.es.md:219]:
```suggestion
La respuesta va a mostrar el estado pendiente del pago y toda la información que necesitas para mostrar al comprador. El valor `transaction_data` te brindará los datos para disponibilizar la opción de pago a través de un código QR. Encontrarás los siguientes atributos:
```

File: guides/online-payments/checkout-api/receiving-payment-by-pix.es.md [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+```html
+<label for="copiar">Copiar Hash:</label>
+<input type="text" id="copiar"  value={qr_code}/>
+```
+
+> NOTE
+>
+> Importante
+>
+> Si necesitas devolver un pago realizado a través de una transferencia Pix, puedes encontrar esta información en [Devoluciones y cancelaciones](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/guides/manage-account/account/cancellations-and-refunds).
```
Comment (romisoto) [inline guides/online-payments/checkout-api/receiving-payment-by-pix.es.md:329]: devolver = reembolsar?

File: guides/snippets/test-integration/pix-intro.en.md [hunk 5]
```diff
@@ -1,14 +1,33 @@
-## Receive payments with Pix
+# Integrate Checkout API payment for Pix
 
-You can offer the option to receive payments instantly with Pix from any bank or digital wallet, either via a **QR code** or a **payment code** where you can set an expiration date for the payment is made.
+You can offer the option to receive payments instantly with Pix (`bank_transfer`) from any bank or digital wallet, either via a **QR code** or a **payment code** where you can set an expiration date for the payment is made.
```
Comment (romisoto) [inline guides/snippets/test-integration/pix-intro.en.md:3]:
```suggestion
You can offer the option to receive payments instantly with Pix (`bank_transfer`) from any bank or digital wallet, either via a **QR code** or a **payment code** where you can set an expiration date for the payment to be made.
```

File: guides/snippets/test-integration/pix-intro.es.md [hunk 6]
```diff
@@ -1,14 +1,34 @@
-## Reciba pagos con Pix
+# Integra Checkout API para pagos con Pix
 
-Puede ofrecer la opción de recibir pagos instantáneamente con Pix desde cualquier banco o billetera digital, ya sea a través de un **código QR** o un **código de pago** donde puede establecer una fecha de vencimiento para que se realice el pago.
+Puede ofrecer la opción de recibir pagos instantáneamente con Pix (`bank_transfer`) desde cualquier banco o billetera digital, ya sea a través de un **código QR** o un **código de pago** donde puede establecer una fecha de vencimiento para que se realice el pago.
```
Comment (romisoto) [inline guides/snippets/test-integration/pix-intro.es.md:3]:
```suggestion
Puedes ofrecer la opción de recibir pagos instantáneamente con Pix (`bank_transfer`) desde cualquier banco o billetera digital, ya sea a través de un **código QR** o un **código de pago** donde puede establecer una fecha de vencimiento para que se realice el pago.
```

**Review by romisoto — CHANGES_REQUESTED** [review]
Te deje algunos comentarios, cualquier cosa si algo no se entiende preguntame!

File: guides/online-payments/checkout-api/other-payment-ways.en.md [hunk 7]
```diff
@@ -3344,13 +3344,12 @@ Both for frontend and backend, we recommend [our libraries](https://www.mercadop
 
 ### Payment Methods
 
-In addition to cards, you can offer other payment choices on your website.
+In addition to cards e Pix, you can offer other payment choices on your website.
```
Comment (romisoto) [inline guides/online-payments/checkout-api/other-payment-ways.en.md:3347]:
```suggestion
In addition to cards and Pix, you can offer other payment choices on your website.
```

File: guides/online-payments/checkout-api/receiving-payment-by-pix.en.md (hunk 1 above)
Comment (romisoto) [inline guides/online-payments/checkout-api/receiving-payment-by-pix.en.md:8]: no consigo hacer funcionar este bookmark. Te funciona?

Comment (hgaldino) [reply guides/online-payments/checkout-api/receiving-payment-by-pix.en.md:8]: Arrumei, Romi!

**Review by romisoto — APPROVED** [review]

**Pull request merged**